<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CookView</class>
 <widget class="QDialog" name="CookView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="all_mission_widget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>441</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>做菜系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>任务列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_mission_btn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>认领菜品</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="my_mission_widget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>311</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我的任务</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="time_count">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前任务</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="current_mission">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>90</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>150</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分钟</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>已用时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_mission_btm">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>完成菜品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="begin_mission_btn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>开始做菜</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_btn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <zorder>all_mission_widget</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>get_mission_btn</zorder>
  <zorder>my_mission_widget</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>current_mission</zorder>
  <zorder>time_count</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>finish_mission_btm</zorder>
  <zorder>begin_mission_btn</zorder>
  <zorder>quit_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
